tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -32,6 +32,8 @@
 RESOURCE_TYPE_BRICK = "brick"
 RESOURCE_TYPE_PEER = "host"
 RESOURCE_TYPE_VOLUME = "volume"
+BRICK_STOPPED = "stopped"
```

Comment:
On L25, replace with "from tendrl.commons.utils import event_utils"

instead of 
 from tendrl.gluster_integration.sds_sync import event_utils

Reply:
i will fix this